generators/app/index.js

```diff
@@ -33,13 +59,24 @@ module.exports = class extends Generator {
           )
         )
       : {};
+    if (this.options.removeOld) {
```

Comment:
I'd prefer we don't add a new option. I think it makes sense to just always remove the old version; and it'd easily fit the 95% use case.